code/modules/mining/machine_redemption.dm

```diff
@@ -93,7 +93,15 @@
 		if(!M || !redemption_mat)
 			return 0
 
-		build_amount = min(build_amount, round(redemption_mat.amount / M))
+		var/smeltable_sheets = round(redemption_mat.amount / M)
+
+		if(!smeltable_sheets)
+			return 0
```

Comment:
This proc is expected to return with a numeric value. I know that the FALSE define's value is zero, but it would look weird. Then again, I did name the proc can_smelt_alloy, which is misleading, so I would rather rename it to smeltable_alloy_amount instead.